tests/integration/suite/scheduler/api/http/errors.go

```diff
@@ -65,57 +65,9 @@ func (e *standardizedErrors) Run(t *testing.T, ctx context.Context) {
 	httpClient := util.HTTPClient(t)
 
 	// Covers apierrors.Empty() job name is empty
-	t.Run("schedule job name is empty", func(t *testing.T) {
```

Comment:
Why do you want to rm this?

Reply:
The diff makes it weird, but the test is further down. https://github.com/cicoyle/dapr/pull/25/files/75164353f93cdc711abef569323594f0af3fe02c#diff-25d4e3f7c84f2ffef0b42a60c664aa8340adc0cc232e1b63db9f79ec56c5285fR115

Reply:
ahh ok